### DISCUSSION TITLE
[QUESTION] - TabPanel is re-rendering on Tab change. How can I prevent re-rendering?

### DISCUSSION THREAD

**Discussion by sujit-baniya** [primefaces/community#2275] [PrimeReact]
I've forked code from website and made some changes

https://stackblitz.com/edit/s5hnpw?file=src%2FApp.jsx

Here the TabPanel is re-rendering.
How can I prevent re-rendering?

**Comment by melloware** [comment]
Well it dynamically switching tabs i would expect it to re-render?  You could try to memoize the panels to see if that helps.
